.github/workflows/publish_branch.yml

```diff
@@ -1,11 +1,10 @@
-name: "publish-branch"
+name: "Publish development branch packages"
 
 on:
-  workflow_dispatch:
+  push:
```

Comment:
Do we really need a snapshot on every push to github? Perhaps it would be better to leave to use `workflow_dispatch` and use that when you need a snapshot?

Reply:
Since we are pushing to our internal registry I think it would be harmless. It's kind tedious when you are integrating your changes (e.g into `aqua-playground` ) and have to trigger the workflow for every commit

Reply:
We decided to leave it as is for now